spec/ruby-progressbar/calculators/length_spec.rb

```diff
@@ -43,6 +43,41 @@ module    Calculators
     expect(length_calculator.length).to eql 99
   end
 
+  it 'if output is a tty, uses it to calculate length in unix environments' do
```

Comment:
It's easy to miss in the diff, but I also moved this spec file from `lib` to `spec`.

Reply:
I think you based your branch on an older commit than what is currently `master`.  That's been fixed on `master` but I'll rebase it, not a big deal.

Reply:
Oh no, sorry, my mistake, I thought I'd fixed it but must not have committed it.  Thanks for the heads up!